PixaPicker/JSONHandler.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  JSONHandler.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 9/26/19.
+//  Copyright © 2019 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+
+class JSONHandler {
+    
+    struct Root : Decodable {
```

Comment:
This is so simple and genius.  I might steal adding these structs internally lol.